lib/led/rgb.js

```diff
@@ -1,12 +1,158 @@
+    write: {
+      writable: true,
+      value: function(colors) {
+        var state = priv.get(this);
+
+        RGB.colors.forEach(function(color, index) {
+          var pin = this.pins[index];
+          var value = colors[color];
+
+          if (state.isAnode) {
+            value = 255 - Board.constrain(value, 0, 255);
+          }
+
+          this.io.analogWrite(pin, value);
+        }, this);
+      }
+    }
+  },
+  PCA9685: {
+    COMMANDS: {
+      value: {
+        PCA9685_MODE1: 0x0,
+        PCA9685_PRESCALE: 0xFE,
+        LED0_ON_L: 0x6
+      }
+    },
+    initialize: {
+      value: function(opts) {
+        this.address = opts.address || 0x40;
+
+        if (!this.board.Drivers[this.address]) {
+          this.io.i2cConfig();
+          this.board.Drivers[this.address] = {
+            initialized: false
+          };
+
+          // Reset
+          this.io.i2cWrite(this.address, [this.COMMANDS.PCA9685_MODE1, 0x0]);
+          // Sleep
+          this.io.i2cWrite(this.address, [this.COMMANDS.PCA9685_MODE1, 0x10]);
+          // Set prescalar
+          this.io.i2cWrite(this.address, [this.COMMANDS.PCA9685_PRESCALE, 0x70]);
+          // Wake up
+          this.io.i2cWrite(this.address, [this.COMMANDS.PCA9685_MODE1, 0x0]);
+          // Wait 5 nanoseconds for restart
+          nanosleep(5);
+          // Auto-increment
+          this.io.i2cWrite(this.address, [this.COMMANDS.PCA9685_MODE1, 0xa1]);
+
+          this.board.Drivers[this.address].initialized = true;
+        }
+
+        RGB.colors.forEach(function(color, index) {
+          var pin = opts.pins[index];
+          this.pins[index] = pin;
+        }, this);
+      }
+    },
+    write: {
+      writable: true,
+      value: function(colors) {
+        var state = priv.get(this);
+
+        RGB.colors.forEach(function(color, index) {
+          var pin = this.pins[index];
+          var value = colors[color];
+          var on, off;
+
+          if (state.isAnode) {
+            value = 255 - Board.constrain(value, 0, 255);
+          }
+
+          on = 0;
+          off = value * 4095 / 255;
+
+          this.io.i2cWrite(this.address, [this.COMMANDS.LED0_ON_L + 4 * pin, on, on >> 8, off, off >> 8]);
+        }, this);
+      }
+    }
+  },
+  BLINKM: {
+    COMMANDS: {
+      value: {
+        GO_TO_RGB_COLOR_NOW: 0x6e,
+        STOP_SCRIPT: 0x6f
+      }
+    },
+    initialize: {
+      value: function(opts) {
+        this.address = opts.address || 0x09;
+
+        if (!this.board.Drivers[this.address]) {
+          this.io.i2cConfig();
+          this.board.Drivers[this.address] = {
+            initialized: false
+          };
+
+          // Stop the current script
+          this.board.io.i2cWrite(this.address, [this.COMMANDS.STOP_SCRIPT]);
+
+          this.board.Drivers[this.address].initialized = true;
+        }
+      }
+    },
+    write: {
+      writable: true,
+      value: function(colors) {
+        this.board.io.i2cWrite(this.address,
```

Comment:
One thing I want to clean up here (and eventually everywhere else, as we go along) is to migrate all occurrences of `this.board.io` to `this.io`